src/contexts/CubeContext.js

```diff
@@ -805,7 +817,7 @@ export const CubeContextProvider = ({ initialCube, cards, children, loadVersionD
     [cube, changes, setChanges],
   );
 
-  const canEdit = user && cube.owner && cube.owner.id === user.id;
+  const canEdit = canUserEdit(cube, user);
```

Comment:
This field also controls other things, such as setting tag colors and editing draft formats. Whether this value was used over an explicit equality check seems to have been somewhat arbitrary in the past. Have you gone through its usages and checked which options it affects? 

The naming note from above also applies here.